validator/testdata/feature_tests/noscript.out

```diff
@@ -83,4 +83,4 @@ feature_tests/noscript.html:65:2 The tag 'video' may only appear as a descendant
 >>              ^~~~~~~~~
 feature_tests/noscript.html:69:13 The parent tag of tag 'noscript enclosure for boilerplate' is 'noscript', but it can only be 'head'. (see https://github.com/ampproject/amphtml/blob/master/spec/amp-boilerplate.md) [DISALLOWED_HTML]
 |  </body>
-|  </html>
\ No newline at end of file
+|  </html>
```

Comment:
One of these days I'd like to solve this... Happen to know where it's happening?